pulsar-sql/presto-pulsar/src/main/java/org/apache/pulsar/sql/presto/PulsarColumnMetadata.java

```diff
@@ -128,9 +117,91 @@ public int hashCode() {
         int result = super.hashCode();
         result = 31 * result + (isInternal ? 1 : 0);
         result = 31 * result + (nameWithCase != null ? nameWithCase.hashCode() : 0);
-        result = 31 * result + Arrays.hashCode(fieldNames);
-        result = 31 * result + Arrays.hashCode(positionIndices);
```

Comment:
Is it necessary to compute the `DecoderExtraInfo`?

Reply:
thanks for the remind, fixed as suggestion.